src/main/kotlin/com/semgrep/idea/lsp/SemgrepLspServerSupportProvider.kt

```diff
@@ -11,7 +11,8 @@ class SemgrepLspServerSupportProvider : LspServerSupportProvider {
         file: VirtualFile,
         serverStarter: LspServerSupportProvider.LspServerStarter
     ) {
-        val installed = SemgrepInstaller.semgrepInstalled()
+        val settingState = AppState.getInstance().lspSettings
+        val installed = if (settingState.useJS) true else SemgrepInstaller.semgrepInstalled()
```

Comment:
is that idiomatic Kotlin?? (vs. `settingsState.useJS || SemgrepInstaller.semgrepInstalled()`)

Reply:
sorry, was code written on a friday, brain was not working